### PR TITLE
Fix/cbl2.5 system test

### PR DESCRIPTION
**Pull request by hrajput89**
#### Fixes #.

- [x] Ran `flake8`
- [x] Ran `run_repo_tests.sh`

#### Changes proposed in this pull request:

- Made changes to System test for optional rebalancing and file logging
- Adding file logging options to all conftests

### CODE REVIEW

File: testsuites/CBLTester/CBL_Functional_tests/conftest.py [hunk 1]
```diff
@@ -490,6 +503,11 @@ def params_from_base_test_setup(request, params_from_base_suite_setup):
     db = None
     cbl_db = None
     if create_db_per_test:
+        if enable_file_logging and liteserv_version >= "2.5.0":
+            cbllog = FileLogging(base_url)
+            cbllog.configure(log_level="verbose", max_rotate_count=2,
```
Comment (sridevi-15) [inline testsuites/CBLTester/CBL_Functional_tests/conftest.py]: can we have something like enable_file_logging=verbose or debug or something else? Whatever is set, it sets log_level? I know you need another verification whether valid log_levels are added or not. We don't really care about other log levels. If it is too hard, you can leave as it is

File: testsuites/CBLTester/topology_specific_tests/multiple_sync_gateways/conftest.py [hunk 2]
```diff
@@ -243,8 +246,6 @@ def params_from_base_suite_setup(request):
             logging_helper.fetch_and_analyze_logs(cluster_config=cluster_config, test_name=request.node.name)
             raise
 
-    cluster.reset(sg_config)
```
Comment (sridevi-15) [inline testsuites/CBLTester/topology_specific_tests/multiple_sync_gateways/conftest.py]: Why is this removed?

File: testsuites/CBLTester/CBL_Functional_tests/conftest.py (hunk 1 above)
Comment (hrajput89) [reply testsuites/CBLTester/CBL_Functional_tests/conftest.py]: basically verbose is enabled only for debug builds. For release build verbose and debug are same. Hence I provided verbose as log level

File: testsuites/CBLTester/topology_specific_tests/multiple_sync_gateways/conftest.py (hunk 2 above)
Comment (hrajput89) [reply testsuites/CBLTester/topology_specific_tests/multiple_sync_gateways/conftest.py]: It's not needed as we reset cluster in test itself. Also cluster reset is just after provisioning; doesn't make any sense.
